frontend/public/components/overview/_build-overview.scss

```diff
@@ -0,0 +1,25 @@
+.build-overview__item {
+  display: flex;
+  flex-direction: column;
+}
+
+.build-overview__item-title {
+  display: flex;
+  justify-content: space-between;
+}
+
+.build-overview__item-reason {
+  border-left: 2px solid $color-pf-red-100;
```

Comment:
I'd suggest using the same variable for this failed color and pipelines. They're currently the same, but we should use a common var so they always match.

https://github.com/openshift/console/blob/master/frontend/public/components/_build-pipeline.scss#L3